components/stream-provider/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "stream-provider",
+  "version": "1.0.0",
+  "description": "App that streams from thr cloud",
+  "main": "dist/stream-provider/src/node/main.js",
```

Comment:
shouldn't these paths be relative to this file location?

Reply:
not sure what you mean. The full path to main.js is 
c:/.../browserd/components/stream-provider/dist/stream-provider/src/node/main.js

Reply:
how does `stream-provider` end up in that path twice?

Reply:
When we run build, the signal-provider folder gets packaged with the shared folder so I have to specify which src to use
![image](https://user-images.githubusercontent.com/10469832/61057320-0fc48b80-a3c3-11e9-89fb-4dd00ba4db4f.png)

Reply:
got it. thanks!